mix.exs

```diff
@@ -10,14 +10,53 @@ defmodule KinesisClient.Mixfile do
       start_permanent: Mix.env() == :prod,
       package: package(),
       description: description(),
+      compilers: [:unused] ++ Mix.compilers(),
+      test_coverage: [tool: ExCoveralls],
       deps: deps(),
       source_url: "https://github.com/uberbrodt/kcl_ex",
-      test_coverage: [tool: ExCoveralls],
       preferred_cli_env: [
         coveralls: :test,
         "coveralls.detail": :test,
         "coveralls.post": :test,
         "coveralls.html": :test
+      ],
+      unused: [
+        ignore: [
```

Comment:
With this many ignores, I'd probably opt for removing the mix_unused lib. Since this repo is meant to be imported as a dependency, it makes sense that most of its API surface would be "unused". 😄